### ISSUE TITLE
WPML / Translation Friendly

### ISSUE THREAD

**Issue by michalzwierz** [kkarpieszuk/wc-price-history#51]
WPML Can't find a string for the message, so cannot be used on multilingual websites :(
or maybe it can be used, but I don't know how?

**Comment by kkarpieszuk** [comment]
Hmm, that should not be hard to add compatibility. I don't have access to WPML now so I can't test if it works. Fortunately, I worked WPML Compatibility team for 8 years ;)

Please add wpml-config.xml file in main plugin directory with given content:

```
<wpml-config>
	<admin-texts>
		<key name="wc_price_history_settings">
			<key name="display_text" />
		</key>
	</admin-texts>
</wpml-config>
```
and deactivate plugin and activate again. Then go to woocommerce > wc price history and save the data (you can update displayed text)

Now if you go to WPML > String Translation the  string should be there available for translation.

Please tell me if it works so I will include this file into release (without it next plugi  updates would remove the file from your directory)

**Comment by kkarpieszuk** [comment]
I have a good news! I just released a new version 1.8.0 which is adressing this issue. Please update the plugin from your wordpress dashboard.
